keys/dalite-prod.key

```diff
@@ -0,0 +1 @@
+{{ DALITE_TARSNAP_KEY }}
```

Comment:
This role doesn't need to know about the different services - the services depend on it, not the other way around. Can you make this more generic, to avoid having to store individual files for each service key, but rather let the service role provide it?